src/mapeo-manager.js

```diff
@@ -43,8 +53,26 @@ export class MapeoManager {
       tables: [projectTable],
       sqlite,
     })
-    this.#storagePath = storagePath
     this.#activeProjects = new Map()
+
+    const filePool =
+      dbFolder !== ':memory:'
+        ? new RandomAccessFilePool(MAX_FILE_DESCRIPTORS)
+        : undefined
+
+    this.#createProjectStorageOpts =
+      dbFolder === ':memory:'
+        ? () => ({
+            dbPath: ':memory:',
+            storage: () => new RAM(),
+          })
```

Comment:
have a feeling that this shouldn't be a function that creates a new RAM instance each time a project is created. is the idea to share the ram instance across all projects too?